runtime/rasdump/javadump.cpp

```diff
@@ -5149,6 +5164,59 @@ JavaCoreDumpWriter::writeCPUinfo(void)
 	return;
 }
 
+void
+JavaCoreDumpWriter::writeCgroupMetrics(void)
+{
+	OMRPORT_ACCESS_FROM_J9PORT(_PortLibrary);
+	BOOLEAN isCgroupSystemAvailable = omrsysinfo_cgroup_is_system_available();
+	if (isCgroupSystemAvailable) {
+		OMRCgroupEntry *entryhead = omrsysinfo_get_cgroup_subsystem_list();
```

Comment:
this should be `const` so it doesn't get changed by accident

Reply:
made it `const`